docs/services/navigation/orientation.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "How Orientation is Measured in Viam"
+linkTitle: "Orientation"
+weight: 10
+type: "docs"
+description: "How Viam reads and utilizes the orientation measurements reported by some models of movement sensor."
```

Comment:
This talks about measurement via movement sensor, but it's not just measured; it's also an input in a lot of places that don't use movement sensor. It can be an argument a user passes, or an output from a motion service function, etc.

Reply:
Exactly! I'm not trying to negate that with this claim, merely provide context, so I don't think any change is needed? Docs checks won't let me make the description any longer

Reply:
This comment was based on this PR removing the other orientation vector page, in which case this page would've become the only place where orientation vectors are discussed. In such a case, this description should be shorter and more general, like
```suggestion
description: "How Viam describes orientation."
```
or something